modules/cassandra/build.gradle

```diff
@@ -8,7 +8,7 @@ configurations.all {
 
 dependencies {
     api project(":database-commons")
-    api "com.datastax.cassandra:cassandra-driver-core:3.10.0"
+    api "com.datastax.oss:java-driver-core:4.17.0"
```

Comment:
We shouldn't update this one because is a transitive dependency and there could be consumers relying on this versions. Using the new `CassandraContainer` implementation, this can be excluded.

Reply:
Ok, I'll revert the changes on the existing implementation and just keepinh the deprecated annotation.